streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -174,13 +222,16 @@ public void kStreamKTableJoin(String kStreamTopic, String kTableTopic) throws Ex
      * KStream record joins to exactly one element in the other KStream
      */
     public void kStreamKStreamJoin(String kStreamTopic1, String kStreamTopic2) throws Exception {
-        CountDownLatch latch = new CountDownLatch(numRecords);
+        CountDownLatch latch = new CountDownLatch(1);
 
         // initialize topics
-        System.out.println("Initializing kStreamTopic " + kStreamTopic1);
-        produce(kStreamTopic1, VALUE_SIZE, "simple-benchmark-produce-kstream-topic1", numRecords, true, numRecords, false);
-        System.out.println("Initializing kStreamTopic " + kStreamTopic2);
-        produce(kStreamTopic2, VALUE_SIZE, "simple-benchmark-produce-kstream-topic2", numRecords, true, numRecords, false);
+        if (loadPhase) {
```

Comment:
At a glance it looks like this block of code is pretty much the same as the code in `kTableKTableJoin` and `kStreamKTableJoin` - can we extract this into a method?